06-improve-model-performance/improvement-overview.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>Link to the documentation and give an explanation for <code>enable_rerank</code> and <code>top_k_retrieved_chunks</code></p><p> </p><p>I would also add something like "These are advanced parameters, for most users you will get good results from leaving these as platform defaults."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>